src/posture_estimator.h

```diff
@@ -89,8 +91,14 @@ struct PoseStatus {
   Pose current_pose;
   Pose pose_changes;
   bool good_posture;
+  bool ideal_pose_set;
```

Comment:
I would advocate combining these into a single variable that could be an enum with possible values of: `Good`, `Bad`, and `Undefined`/`Unset`. That means only checking on variable instead of two. It also means we could expand it easily if more states are required for a `PoseStatus`.

Reply:
Although you make a valid point @miklasr, the signal related to this for the GUI emits "poseStatus" which contains both of these variables so we have access to both variables no problem. So not sure if we need to change this? Your comment still makes sense just pointing out it is not necessary.

Reply:
Yeah, it's not necessary but it just makes things a bit clearer -- it's easier and safer to check one variable instead of two, without adding particular overheads to refactor this (I think).